app.js

```diff
@@ -0,0 +1,15 @@
+const modal = document.getElementById("modal");
+const modalBtn = document.querySelectorAll(".modalBtn");
+const closeBtn = document.getElementById("closeModal");
+
+modalBtnList = [...modalBtn];
+
```

Comment:
You forgot to remove this variable

Reply:
Oh, thanks. I was trying to iterate through nodelist but wasn't able to so I found this on StackOverflow. Now when you pointed out, I saw that I haven't used it. Thanks again :)